mriqc/bin/fs2gif.py

```diff
@@ -121,7 +121,7 @@ def main():
             modnii = op.join(tmp_sub, "%s.nii.gz" % subid)
             ref_file = op.join(tmp_sub, "%s.mgz" % subid)
             img = nb.load(niifile)
-            data = exposure.equalize_adapthist(img.get_data(), clip_limit=0.03)
+            data = exposure.equalize_adapthist(img.get_fdata(), clip_limit=0.03)
             nb.Nifti1Image(data, img.affine, img.header).to_filename(modnii)
             sp.call(["mri_convert", modnii, ref_file], cwd=tmp_sub)
```

Comment:
Similarly (two part, since this is split by a deletion):

```suggestion
            nb.MGHImage(data, img.affine, img.header).to_filename(ref_file)
```

Feel free to ignore these, too...